src/dlgTriggerEditor.cpp

```diff
@@ -4082,7 +4082,7 @@ void dlgTriggerEditor::saveAlias()
     mpAliasMainArea->trimName();
     QString name = mpAliasMainArea->lineEdit_alias_name->text();
     QString regex = mpAliasMainArea->lineEdit_alias_pattern->text();
-    if ((name.size() < 1) || (name == "New Alias")) {
+    if ((name.size() < 1) || (name == tr("New alias"))) {
```

Comment:
BTW `(name.isEmpty())` better signals intent rather than `(name.size() < 1)`

Reply:
out of scope

Reply:
Tackled as part of #2638